### ISSUE TITLE
exported tiff seen as dmg 

### ISSUE THREAD

**Issue by lromano** [darktable-org/darktable#14126]
### Checklist

- [X] I have checked [GitHub Issues](https://github.com/darktable-org/darktable/issues?q=is%3Aissue) and my problem is **not** described there.

### Describe the bug

When exporting a finalized image with the export module, I noticed that several printing websites like whitewall or cewe were 
refusing the image.

In whitewall, the returned error is clear : the file is a dng.

I tried to import it in Photoshop to verify this and indeed, it was opened via Camera Raw, which is a plugin used by Photoshop to deal with raw images.

It doesn't matter if I select 8 or 16 bits, include a specific icc profile or not, choose compression or not, the outcome is the same.

This does not seem to be normal and as a result, the only other option is exporting a jpeg, which is not ideal.

### Steps to reproduce

Go to lighttable, double click or click 'd' to go in the darkroom, perform any random edit.

Go back to lighttable, select the image you want to export.

Go to the export module and select tiff, 8 or 16 bits, compressed or uncompressed.

Open a browser, try to upload the file to whitewall via the button 'upload photos'
navigating to https://www.whitewall.com/eu/

Notice that the file refuses to upload. If you check the browser developer tools / network tabs, you'll see the error. It complains about the format being a dng

You'll get the same error if you try to upload your tiff here : 
https://www.cewe.be/fr/deco-murale/configurateur.html

### Expected behavior

darktable should export a file fully recognized as a tiff

### Logfile | Screenshot | Screencast

Example of export config giving the issue : 
![reglages-tiff-darktable](https://user-images.githubusercontent.com/1656755/230202320-5d3f2d4f-d371-438a-8aad-2363ed264b98.png)
white wall website error response :
![erreur-whitewall-dng](https://user-images.githubusercontent.com/1656755/230202340-12ef883b-5c5b-4b4c-976a-10a39e37aac8.png)

### Where did you install darktable from?

darktable.org

### darktable version

4.2.1

### What OS are you using?

Linux

### What is the version of your OS?

Linux 5.10.176-2-MANJARO

### Describe your system?

Intel(R) Core(TM) i5-9600K CPU @ 3.70GHz
16gb ram
Nvidia RTX 2700

### Are you using OpenCL GPU in darktable?

No

**Comment by ralfbrown** [comment]
Please upload a sample file for analysis.

As it happens, DNG files **are** TIFF files.  TIFF is a container format and can contain images in multiple different encodings, including JPEG and un-demosaiced raw.  Every camera RAW format I'm familiar with except .CR3 is in fact a TIFF container, and the "JPEG" files your post-1998 camera creates are also TIFF containers, but with a small header prepended so that older software recognizes them as JFIF-format JPEGs.

**Comment by parafin** [comment]
It’s probably about TIFF tags dt writes to the exported files.

**Comment by kmilos** [comment]
Yeah, that "wrong image format: DNG" message is completely useless.

Can you try perhaps removing this one suspect tag by running `exiftool -TIFF-EPStandardID= foo.tif`?

**Comment by kmilos** [comment]
Also, if you don't want to share an image, please at least share the output of `exiftool -a -u -s -g1 -w txt foo.tif` for a "failing" case.

**Comment by lromano** [comment]
Hello, here is a sample file, I had to put it in a zip because of github validation rules + the output of the file
[DSC00146.tif.zip](https://github.com/darktable-org/darktable/files/11173525/DSC00146.tif.zip)
[DSC00146.txt](https://github.com/darktable-org/darktable/files/11173532/DSC00146.txt)

**Comment by kmilos** [comment]
Ah, so this was actually an imported DNG created by Adobe Lightroom, and there are a bunch of DNG/Lightroom tags left over, hard to tell which one exactly throws off those apps. In addition, there is XMP metadata from Lightroom, also hard to tell if we should be messing w/ it...

It could be as simple as
```
---- XMP-dc ----
Format                          : image/dng
```
so you can try `exiftool -XMP-dc:Format= foo.tif`...

Otherwise, you can also try removing these DNG specific tags one by one (w/ `exiftool -TagName= foo.tif`):
```
UniqueCameraModel        
CameraCalibration1       
CameraCalibration2       
AnalogBalance            
AsShotNeutral            
BaselineExposure         
BaselineNoise            
BaselineSharpness        
LinearResponseLimit      
ShadowScale              
CameraCalibrationSig     
PreviewApplicationName   
PreviewApplicationVersion
PreviewSettingsDigest    
PreviewColorSpace        
PreviewDateTime          
NoiseProfile             
NewRawImageDigest        
```

Once we find what it is, making the code change should be straightforward (famous last words)..

**Comment by lromano** [comment]
Hello everyone, seems that kmilos found the culprit. 

Indeed, this image is part of old dng files convert by Lightroom from  my camera. I suppose that the program added a bunch of tags at this moment.

to verify that, I tried to upload  an image, this time directly coming from a raw imported straight into darktable; and then, no issues to upload te file.

I also tried to execute this command that you recommended, into the problematic dng from Lightroom :
**exiftool -XMP-dc:Format= foo.tif**

And then, no problems to reupload, issue is fixed.

Here is the modified sample file :
[sample.tif.zip](https://github.com/darktable-org/darktable/files/11182777/sample.tif.zip)
